src/ceph-daemon

```diff
@@ -380,6 +381,31 @@ def install_base_units(fsid):
         subprocess.check_output(['systemctl', 'enable', 'ceph-%s.target' % fsid])
         subprocess.check_output(['systemctl', 'start', 'ceph-%s.target' % fsid])
 
+    # logrotate for the cluster
+    with open(args.logrotate_dir + '/ceph-%s' % fsid, 'w') as f:
+        """
+        This is a bit sloppy in that the killall/pkill will touch all ceph daemons
+        in all containers, but I don't see an elegant way to send SIGHUP *just* to
+        the daemons for this cluster.  (1) systemd kill -s will get the signal to
+        podman, but podman will exit.  (2) podman kill will get the signal to the
+        first child (bash), but that isn't the ceph daemon.  This is simpler and
+        should be harmless.
+        """
+        f.write("""# created by ceph-daemon
+/var/log/ceph/%s/*.log {
+    rotate 7
+    daily
+    compress
+    sharedscripts
+    postrotate
+        killall -q -1 ceph-mon ceph-mgr ceph-mds ceph-osd ceph-fuse radosgw || pkill -1 -x "ceph-mon|ceph-mgr|ceph-mds|ceph-osd|ceph-fuse|radosgw" || true
```

Comment:
rbd-mirror? rbd-nbd? anything else I'm forgetting? I guess this is just copied from the existing logrotate conf, nevermind